### DISCUSSION TITLE
Version 0.30.0 is out!

### DISCUSSION THREAD

**Discussion by tcharding** [rust-bitcoin/rust-bitcoin#1724] [Announcements]
Out now, get it while its hot
- https://crates.io/crates/bitcoin/0.30.0
- https://rust-bitcoin.org/blog/release-0.30.0/

**Comment by tcharding** [comment]
And well done team, epic release if I don't say so myself.

**Reply by DanGould** [reply]
awesome post & guidance too

**Reply by Kixunil** [reply]
Happy to hear it was helpful!

**Reply by tcharding** [reply]
Thanks man, nice to hear.
